src/executables/master.c

```diff
@@ -1130,6 +1132,17 @@ main (int argc, char **argv)
   const char *msg_format;
   bool util_config_ret;
 
+  // TODO : When no non-HA server exists in HA environment, server_monitor should not be initialized.
+  //        In this issue, server_monitor is initialized only when HA is disabled. Once all sub-tasks of
+  //        CBRD-24741 are done, this condition will be removed. And server_monitor will be initialized wh
+  //        en first non-HA server is started in HA environment. (server_monitor will be finalized when last
+  //        non-HA server is stopped in HA environment.)
+  if (HA_DISABLED ())
+    {
+    // *INDENT-OFF*
+      server_monitor& server_monitoring_manager = server_monitor::get_instance();
+    // *INDENT-ON*
```

Comment:
```suggestion
      // *INDENT-OFF*
      server_monitor& server_monitoring_manager = server_monitor::get_instance();
      // *INDENT-ON*
```
시작 지점 맞춰주는게 좋을 것 같아요.